### ISSUE TITLE
Opening Source file oddly causes tests to begin failing.

### ISSUE THREAD

**Issue by Barryrowe** [wallabyjs/public#888]
### Issue description or question

In VS Code (Insiders), if I open a project, and start wallaby all of the tests pass. As soon as I open up a source file, the file loses all wallaby gutter highlighting and the tests begin failing.

This is an Angular2 project, and the errors are like:

```
  ​​​​The CorpCommon Auth API should call cchttp with the correct parameters when authenticating​​​​ ​​​[12 ms]​​​​
    ​​​​
    ​​Error: Invalid provider for the NgModule 'DynamicTestModule' - only instances of Provider and Type are allowed, got: [BaseRequestOptions, MockBackend, [object Object], [object Object], CCUtil, CCHttp, CCStorage, ?undefined?, ...]​​
```

It's odd to me that opening the file would cause this, I don't know if there's somehow a conflict between the VS code tsc compilation and wallaby's tsc compilation. I'm hoping it's not an issue with wallaby but something with angular2.2.3, but I can't narrow it down really, and when all tests pass without opening a file, it seems odd.

I even cloned my project to a new folder, npm installed, opened and started tests to try and make sure there was not cache issue. The tests pass, until I open a source file in the editor (no changes are even made).

I can zip the project up if needed, as it's internal and not publicly available.

### Wallaby.js configuration file

```javascript
var wallabyWebpack = require('wallaby-webpack');

var webpackPostprocessor = wallabyWebpack({
  entryPatterns: [
    'spec-bundle.js',
    'src/**/*spec.js'
  ]
});

module.exports = function (w) {

  return {
    files: [
      {pattern: 'spec-bundle.js', load: false},
      {pattern: 'src/**/*.ts', load: false},
      {pattern: 'src/**/*spec.ts', ignore: true},
      {pattern: 'src/**/*d.ts', ignore: true}
    ],

    tests: [
      {pattern: 'src/**/*spec.ts', load: false}
    ],

    testFramework: "jasmine",

    compilers: {
      '**/*.ts': w.compilers.typeScript({
        emitDecoratorMetadata: true,
        experimentalDecorators: true
      })
    },

    postprocessor: webpackPostprocessor,

    bootstrap: function () {
      window.__moduleBundler.loadTests();
    }
  };
};
```

### Code editor or IDE name and version

Visual Studio Code v1.8.0-insider   (2bd85b8)

### OS name and version

Windows 7

**Comment by Barryrowe** [comment]
Also I am on version 1.0.28 of the plugin. I'm awaiting approval to renew my license, so I can't get the latest updates at this time.

**Comment by ArtemGovorov** [comment]
> Also I am on version 1.0.28 of the plugin.

> Visual Studio Code v1.8.0-insider (2bd85b8)
	
There's [this known issue with the v1.8.0-insider](https://github.com/Microsoft/vscode/issues/16019) (VS Code had introduced some changes to the events they raise via the extension API), the issue was fixed in the plugin version 1.0.31.
You may switch to Visual Studio Code 1.7.2 for now before you can upgrade to 1.0.31.
